code/game/objects/items/devices/flash.dm

```diff
@@ -78,15 +78,15 @@
 
 		var/safety = M:eyecheck(TRUE)
 		if(safety <= 0)
-			M.Weaken(10)
 			flick("e_flash", M.flash)
 				//Vaurca damage 15/01/16
 			var/mob/living/carbon/human/H = M
 			if(isvaurca(H))
 				var/obj/item/organ/eyes/E = H.get_eyes()
 				if(!E)
 					return
-				usr << span("alert", "Your eyes burn with the intense light of the flash!.")
+				usr << span("alert", "Your eyes burn with the intense light of the flash. You keel over, your antennae twitching as you lay in a daze.")

```

Comment:
I don't see this as really being necessary? Verbosity is not really desirable in attack logs.

Reply:
This is only what Vaurca see, it's mostly additional fluff to communicate to the bugperson "oh shit, you're blinded and stunned."

Reply:
And I still do not see it as necessary. Attack logs should be simple, especially for so mundane an item.